lib/constable/application.ex

```diff
@@ -1,27 +1,28 @@
 defmodule Constable.Application do
   use Application
 
-  # See http://elixir-lang.org/docs/stable/elixir/Application.html
+  # See https://hexdocs.pm/elixir/Application.html
   # for more information on OTP Applications
   def start(_type, _args) do
-    import Supervisor.Spec, warn: false
+    import Supervisor.Spec
 
     unless Mix.env == "production" do
       Envy.auto_load
     end
 
     setup_dependencies()
 
+    # Define workers and child supervisors to be supervised
```

Comment:
I don't understand the full ramifications of changing from `worker` to `supervisor` in this section, and am somewhat blindly following the new app template/style. I assume this was an elixir or phoenix change which enables this to be ok...

Reply:
Yeah, a quick glance at what ecto brings in via `use Ecto.Repo` (which we use in `Constable.Repo`) shows that it's a supervisor. So I'm guessing that, it either changed from a single process to being a supervisor or using `worker` here before was incorrect. 

Looking at `Phoenix.Endpoint`, the same thing is true. So I think the change is ok.